save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -318,7 +321,7 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
                         .decodeFromJsonString()
             setState {
                 executionDto = executionDtoFromBackend
-                status = props.status
+                filters.status = props.status
```

Comment:
You either create a new filter instance with constructor, or with `copy`

Reply:
Done, but this leads to uncorrected table output when the page loads
Returned to the first version

Reply:
Still didn't get your point

Reply:
Done